### ISSUE TITLE
When I export my blockmodel it always shows up as a missing texture block. (pls fix)

### ISSUE THREAD

**Issue by ZayPlayz** [JannisX11/blockbench#167]
[!] Before you report an issue, check if if has already been reported. Also check if you are using the latest version of Blockbench.
[!] Prefix the title if your issue with [Issue] or [Suggestion]
[!] If you are having an issue, please press "Ctrl + Shift + i" and click on console. If there are any error messages, please attach a screenshot of it to your issue.

Supply the following information:
- Detailed description of your issue
- Blockbench variant (Webapp/Desktop)
- Operating System

**Comment by Francy-Chan** [comment]
It's probably because you let Auto UV turned on, and then, you didn't put an texture in all sides of the block.
For example:
 `[...]`
`"faces": {`
`       "north": {"uv": [9.5, 0.5, 6.5, 1]},`
`	"east": {"uv": [11, 0.5, 10.5, 1]},`
`	"south": {"uv": [6.5, 0.5, 9.5, 1]},`
`	"west": {"uv": [10.5, 0.5, 11, 1]},`
`	"up": {"uv": [6.5, 10.5, 9.5, 11]},`
`	"down": {"uv": [6.5, 10.5, 9.5, 11]}`
`}[...]`
You're like, saying where in the texture to show, but what texture?
Simply put your texture in there like this (or remove the lines, if you want them to not have an texture):
`[...]`
`"faces": {`
`	"north": {"uv": [9.5, 0.5, 6.5, 1], "texture": "#0"},`
`	"east": {"uv": [11, 0.5, 10.5, 1], "texture": "#0"}`
`}[...]`

Hope it helps. (If not, can you attach your .json file?).

**Comment by JannisX11** [comment]
The problem was most likely missing textures on the model. The original behavior was to replace it with a pseudo texture, which would result in a black and purple texture and would prevent the model from breaking entirely. This behavior has been re-added in 2.0.2.

**Comment by ZayPlayz** [comment]
Oh ok, that was the problem, Thank you Francy-Chan and Jannis. That was really helpful. :D
